source/administration/bucket-replication.rst

```diff
@@ -272,6 +272,11 @@ workers operating on that queue. MinIO continuously works to replicate and
 remove objects from the queue while scanning for new unreplicated objects to
 add to the queue. 
 
+.. versionchanged:: RELEASE.2022-08-11T04-37-28Z
+
+   Failed or pending replications requeue automatically when performing a list or any ``GET`` or ``HEAD`` API method. 
```

Comment:
Q: does this just mean they get tossed into the queue, or do we prioritize them to the front of the queue? cc @poornas

Reply:
Thanks @poornas.
Will merge as is, then.